ui/app/components/clients/page/overview.ts

```diff
@@ -5,8 +5,47 @@
 
 import ActivityComponent from '../activity';
 import { service } from '@ember/service';
+import { ByNamespaceClients, MountClients } from 'core/utils/client-count-utils';
+import { sanitizePath } from 'core/utils/sanitize-path';
 import type FlagsService from 'vault/services/flags';
+import type NamespaceService from 'vault/services/namespace';
 
 export default class ClientsOverviewPageComponent extends ActivityComponent {
   @service declare readonly flags: FlagsService;
+  @service declare readonly namespace: NamespaceService;
+
+  get showNewClientAttribution(): boolean {
+    return this.args.activity.byMonth.length === 1;
+  }
+
+  get newClientNamespaceAttribution(): ByNamespaceClients[] | undefined {
+    if (this.showNewClientAttribution) {
+      return this.args.activity.byMonth[0]?.namespaces;
+    }
+    return;
+  }
+
+  get newClientMountAttribution(): MountClients[] | undefined {
+    if (this.showNewClientAttribution) {
+      const currentNs = this.namespace.currentNamespace;
+      const nsLabel = sanitizePath(this.args.namespace || currentNs || 'root');
+      const singleNamespace = this.newClientNamespaceAttribution?.find(
+        (namespace) => sanitizePath(namespace.label) === nsLabel
+      );
+      return singleNamespace?.mounts;
+    }
+    return;
+  }
```

Comment:
I might be missing it - but I don't see these getters used anywhere? Are they leftover from the side by side attributions that we decided to remove?

Reply:
yup! Thanks 😄